docs/developer/non-admin-controller.md

```diff
@@ -0,0 +1,48 @@
+# OADP NonAdminBackup Controller
+
+The OADP NonAdminBackup Kubernetes controller manages the lifecycle of custom resources called `NonAdminBackup`. Let's break down the code and understand its functionality.
+
+## Purpose
+
+The primary goal of this controller is to allow users with limited permissions (non-admins) to trigger and manage Velero backups within their namespaces. It acts as an intermediary between the non-admin user and Velero, ensuring that backups are created and managed securely within the defined boundaries.
```

Comment:
?
`Within the namespaces to which they have access to`

Reply:
this sounds ai'ish, talk about everything and nothing:
`ensuring that backups are created and managed securely within the defined boundaries.`